docs/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>API documentation</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <style>
+         body {
+             margin: 0;
+             padding: 0;
+         }
+        </style>
+    </head>
+    <body>
+        <redoc spec-url='https://raw.githubusercontent.com/sul-dlss/preservation_catalog/master/openapi.yml'></redoc>
+        <script src="https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.17/bundles/redoc.standalone.js"> </script>
+    </body>
+</html>
```

Comment:
Is this autogen, or required to be provided?  If former, it shouldn't be checked in.  I assume it's the latter. ...

Reply:
Not auto-generated and is required for API docs like https://sul-dlss.github.io/sdr-api/